src/functionalTest/java/uk/gov/hmcts/reform/pip/account/management/controllers/AuditCreationTest.java

```diff
@@ -0,0 +1,150 @@
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ThreadLocalRandom;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.OK;
+
+@ExtendWith(SpringExtension.class)
+@ActiveProfiles(profiles = "functional")
+@SpringBootTest(classes = {OAuthClient.class})
+class AuditCreationTest extends FunctionalTestBase {
+    private static final UUID USER_ID = UUID.randomUUID();
+    private static final String TEST_EMAIL_PREFIX = String.format(
+        "pip-am-test-email-%s", ThreadLocalRandom.current().nextInt(1000, 9999));
+    private static final String TEST_EMAIL = TEST_EMAIL_PREFIX + "@justice.gov.uk";
+    private static final String ROLES = "SYSTEM_ADMIN";
+    private static final String USER_PROVENANCE = "PI_AAD";
+    private static final String ACTION = "PUBLICATION_UPLOAD";
+    private static final String DETAILS = "Publication with artefact id %s successfully uploaded";
+    private static final Integer PAGE_NUMBER = 0;
+    private static final Integer PAGE_SIZE = 2;
+
+    private static final String AUDIT_URL = "/audit";
+    private static final String GET_AUDIT_URL = "/audit/%s";
+    private static final String TESTING_SUPPORT_APPLICATION_URL = "/testing-support/audit/";
+    private static final String BEARER = "Bearer ";
+
+    private Map<String, String> bearer;
+
+    @BeforeAll
+    public void startUp() {
+        bearer = Map.of(HttpHeaders.AUTHORIZATION, BEARER + accessToken);
+    }
+
+    @AfterAll
+    public void teardown() {
+        doDeleteRequest(TESTING_SUPPORT_APPLICATION_URL + TEST_EMAIL_PREFIX, bearer);
+    }
+
+    private AuditLog createAuditLog() {
+
+        UUID artefactId = UUID.randomUUID();
+        String responseBody = """
+            {
+                "userId": "%s",
+                "userEmail": "%s",
+                "roles": "%s",
+                "userProvenance": "%s",
+                "action": "%s",
+                "details": "%s"
+            }
+            """.formatted(USER_ID, TEST_EMAIL, ROLES, USER_PROVENANCE, ACTION, String.format(DETAILS, artefactId));
+
+        Response response = doPostRequest(AUDIT_URL, bearer, responseBody);
+
+        assertThat(response.getStatusCode()).isEqualTo(OK.value());
+
+        return response.getBody().as(AuditLog.class);
+    }
+
+    @Test
+    void shouldBeAbleToCreateAndGetAnAuditRecord() {
+        AuditLog auditLog = createAuditLog();
+
+        Response getResponse = doGetRequest(String.format(GET_AUDIT_URL, auditLog.getId()), bearer);
+        assertThat(getResponse.getStatusCode()).isEqualTo(OK.value());
+
+        AuditLog retrievedAuditLog = getResponse.getBody().as(AuditLog.class);
+        assertThat(retrievedAuditLog.getId()).isEqualTo(auditLog.getId());
+        assertThat(retrievedAuditLog.getUserEmail()).isEqualTo(auditLog.getUserEmail());
+    }
+
+    @Test
```

Comment:
I think we'll run into the issues with the below when it runs on the master pipeline due to master linking up to the Staging DB and the numbers below not matching.

To fix this, I'd suggest using the functionality in the recently merged PUB-1909, to filter based on email alongside the page sizes below to ensure it will always return a consistent result